### PR TITLE
Edit/new commands table

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

Related to issue #535 

### Please describe the goal of this pull request and the changes that were made.

Update list of commands and reformat as table

### Additional Comments

@geealbers I did my best to interpret the command information from the README and our conversation with Matthew. I didn't include the subtags because that felt like too much information from this page, but if you would like them added, then I would love advice on how to best format them.

### CODE REVIEW

File: content/docs-v1/quire-commands.md [hunk 1]
```diff
@@ -15,46 +15,60 @@ Quire commands are typically run from Terminal on a Mac and PowerShell on a PC.
 
 ## Start and Preview Projects
 
-`quire new project-name`
-: Create a new Quire project named `project-name` in the current directory. The name can be anything, but shouldn’t contain spaces or special characters.
+| Command       | Action   | Notes              |
+| ------------- | -------- | ------------------ |
+| **`quire new my-project`** | Create a new Quire project with the default starter content | In this instance the new project is named `my-project` but the name can be anything as long as it doesn't contain spaces or special characters. A folder containing the project files for your new project will be created in your project's home directory. |
```
Comment (geealbers) [inline content/docs-v1/quire-commands.md:20]: > A folder containing the project files for your new project will be created in your project's home directory.

Not sure I follow. Should it be "your _computer's_ home directory"?

File: content/docs-v1/quire-commands.md [hunk 2]
```diff
@@ -15,46 +15,60 @@ Quire commands are typically run from Terminal on a Mac and PowerShell on a PC.
...
+| **`quire configure`** | Edit the Quire CLI configuration | |
+| ** `quire preview pdf`** | Preview your Quire publication as a PDF |
+| **`quire preview epub`** | Preview your Quire publication as an EPUB |
+| **`quire server`** | Start a local web server to serve a previously built Quire site | |
+| **`quire version`** | Set the Quire version you want to use when running commands | This will be useful when switching between old and new Quire projects. |
+| **`quire version install`** | Install a different version of Quire | This will be useful when updating Quire or switching between projects built with different versions. |
+| **`quire version list`** | List installed versions Quire | This command does not include legacy versions of Quire. |
+| **`quire version prune`** | Remove outdated versions of Quire | This command does not include legacy versions of Quire. |
+| **`quire version remove`** | Uninstall versions of Quire | You can specify which version of Quire you would like to remove. |
```
Comment (geealbers) [inline content/docs-v1/quire-commands.md:61]: I think we should remove these, as they're more in the weeds and probably the furthest out from being implemented (if I had to guess)

File: content/docs-v1/quire-commands.md [hunk 3]
```diff
@@ -15,46 +15,60 @@ Quire commands are typically run from Terminal on a Mac and PowerShell on a PC.
...
-: Development use only. Log info about current project.
+| Command       | Action   | Notes              |
+| ------------- | -------- | ------------------ |
+| **`quire --version`** | List Quire version number | This is useful for troubleshooting and confirming updates. |
+| **`quire --help`** | Provide list of Quire commands | This is useful if you need a reminder of commands. |
 
-## Helpful Non-Quire commands
 
-There are some additional commands that you will use that are not specific to Quire.
+## Coming Soon
```
Comment (geealbers) [inline content/docs-v1/quire-commands.md:46]: Maybe "Not Yet Implemented"? Or something more tempered.

**Review by geealbers — APPROVED** [review]
Some notes and such. Feel free to merge when you're ready.

**Comment by Erin-Cecele** [comment]
@geealbers Some of these commands needs to be fleshed out a bit better in terms of the "notes" field, but I will merge this current version, and we can review again later.

**Pull request merged**